dashboard/app/views/pd/fit_weekend1819_registration_mailer/confirmation.html.haml

```diff
@@ -8,15 +8,19 @@
 
 - if @registration.accepted?
   We look forward to meeting you this summer! You will receive more information about
-  travel approximately six weeks before the event. In the meantime, please visit
-  - info_link = "code.org/educate/teachercon/#{@registration.pd_application.fit_workshop.location_city.downcase}"
-  = link_to(info_link, "https://#{info_link}")
+  travel approximately six weeks before the event. In the meantime, please contact
+  = link_to("facilitators@code.org", "mailto:facilitators@code.org")
+  with any questions.
 - else
   We noticed you responded that you’re not able to attend your assigned FiT weekend.
   Please contact us at
-  = link_to("facilitator@code.org", "mailto:facilitator@code.org")
+  = link_to("facilitators@code.org", "mailto:facilitators@code.org")
   so that we can assist you.
 
+%p
+  Thank you,
+  %br
+  Sarah
```

Comment:
I assume we don't want both "Sarah" and "Sarah Fairweather" here and below

Reply:
We do - its by design